components/collection/activity/events/Events.vue

```diff
@@ -27,11 +27,14 @@
 
     <template #rows="{ variant }">
       <EventRow
-        v-for="(event, i) in displayedEvents"
+        v-for="(event, i) in displayedEvents.slice(0, displayedEvents.length)"
```

Comment:
Is there anything different compared to the previous one? Copy an array?
```suggestion
        v-for="(event, i) in displayedEvents
```

Reply:
>Copy an array?

not taking the last array element

the table of events was not loading all the way
point is to render it 1 before the last event row

suggestions on how to do it better are welcome
before:
```
<EventRow />
<EventRow />
...
<div ref="sentinal"/>
```

After:
```
<EventRow />
<EventRow />
...
<EventRow />
<div ref="sentinal"/>
<EventRow/>

Reply:
> > Is there anything different compared to the previous one? Copy an array?
> 
> not taking the last array element
> 

However, it does not do as you expected.  

<img width="338" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/3e84cdef-9431-4757-b611-dca6d642a49d">

Reply:
of course, what an embarrassing mistake
fixed.
Thank you